### PR TITLE
Draft send-announcement docs

### PR DESCRIPTION
**Pull request by timtheisen**
We could probably have a note that python-gnupg is only in EPEL.

Feel free to mod this. I'll see you on Wednesday.

### CODE REVIEW

**Comment by matyasselmeci** [comment]
Having EPEL is pretty much assumed, no?

File: docs/release/sending-announcements.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+| `--dry-run`                     | Use this option until you are ready to actually send the message                                             |
+| `--cert <FILE>`                 | File that contains your OSG User Certificate                                                                 |
+| `--key <FILE>`                  | File that contains your Private Key for your OSG User Certificate                                            |
+| `--no-sign`                     | Don't GPG sign the message (release only)                                                                    |
+| `--type production`             | Not a test message                                                                                           |
+| `--message <FILE>`              | File containing your message                                                                                 |
+| `--subject <EMAIL SUBJECT>`     | The subject of your message                                                                                  |
+| `--recipients <LIST OF EMAILS>` | (optional) List of recipient email addresses, must have at least one                                         |
```
Comment (brianhlin) [inline docs/release/sending-announcements.md]: I'm pretty sure this is optional, right?

Comment (matyasselmeci) [reply docs/release/sending-announcements.md]: If it's moved there, a link would be nice.

**Comment by brianhlin** [comment]
Yea, I think a link from the cutting a release pages would be appropriate

**Comment by matyasselmeci** [comment]
If parts of this doc are moved to a page on the operations repo, there should be a link from here to that page.

File: docs/release/sending-announcements.md (hunk 1 above)
Comment (timtheisen) [reply docs/release/sending-announcements.md]: You need at least one recipient. It is not optional.

Comment (brianhlin) [reply docs/release/sending-announcements.md]: Fixed the doc but this seems like strange behavior. I would think one of `--oim-recipients` and `--recipients` is required.

**Comment by brianhlin** [comment]
This isn't important for the ticket but can we get the tooling installed on osghost so we can send emails without having to worry about the installation process each time?

**Comment by timtheisen** [comment]
I can make sure that osghost has the prerequisites. However, we still need to get the SPF record updated.

**Comment by timtheisen** [comment]
osghost is an EL6 host. So, no go.

**Comment by timtheisen** [comment]
This work was placed in the opensciencegrid/operations repository. Closing.

**Pull request closed without merging**